README.md

```diff
@@ -1 +1,48 @@
-#Taxi-service
\ No newline at end of file
+# Taxi service
+
+This basic web application simulates structure of a taxi service. It allows an
+authenticated user to view and change contents of a taxi service database.
+
+### Capabilities:
+
+- authenticate existing *drivers*;
+- add/register *driver*;
+- display *drivers* and delete them;
+- add new *car manufacturer*;
+- display *car manufacturers* and delete them;
+- add new *car*;
+- display *cars* and delete them;
+- assign *driver* to *car*;
+- display list of *cars* that current authenticated driver is assigned to.
+
+-----------------------------------  
+
+### Implementation features:
+
+- 3-tier architecture (DAO, services, controllers).
+- cars, drivers and car manufacturers are represented by corresponding models
+- taxi_service [schema](Https://i.imgur.com/gw1qj0C.png)
```

Comment:
I'd call this paragraph `Implementation details` and remove last two as features, they aren't